test/linux64/largesectcount.jl

```diff
@@ -0,0 +1,5 @@
+open("test.S", "w") do f
+    for i = 0:0xffff
+        print(f, ".section .text$i\nfoo$i:\nret\n")
+    end
+end
```

Comment:
Is this file actually used for anything?

Reply:
I think it's supposed to be used to generate `largesectcount.S` but it seems to be outputting to `test.S` instead.
Would be nice if it compiled it to `largesectcount.o` as well.